models/aeif_cond_beta_multisynapse.h

```diff
@@ -208,14 +209,62 @@ class aeif_cond_beta_multisynapse : public Archiving_Node
   void get_status( DictionaryDatum& ) const;
   void set_status( const DictionaryDatum& );
 
+  static inline Name
+  get_g_receptor_name( size_t receptor )
+  {
+    std::stringstream receptor_name;
+    receptor_name << "g_" << receptor + 1;
+    return Name( receptor_name.str() );
+  };
+
+  //! Class that reads out state vector elements, used by UniversalDataLogger
+  class DataAccessFunctor
+  {
+    aeif_cond_beta_multisynapse* parent_;
+    size_t elem_;
+
+  public:
+    DataAccessFunctor( aeif_cond_beta_multisynapse& n, size_t elem )
+      : elem_( elem )
+    {
+      set_parent( &n );
+    };
+
+    DataAccessFunctor( aeif_cond_beta_multisynapse* n, size_t elem )
+      : elem_( elem )
+    {
+      set_parent( n );
+    };
+
+    void
+    set_parent( aeif_cond_beta_multisynapse* parent )
```

Comment:
Drop this method and put the single statement into all places calling this function. And if it were to exist further, it ought to be private!

Reply:
Done in eb41277